clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml

```diff
@@ -15,7 +15,7 @@
   In: project(${arg0})
   Out: |
    project(${arg0})
-   set(CMAKE_CXX_FLAGS "\${CMAKE_CXX_FLAGS} -fsycl")
+   set(CMAKE_CXX_FLAGS "\${CMAKE_CXX_FLAGS} -fsycl -DMKL_ILP64")
```

Comment:
```suggestion
   set(CMAKE_CXX_FLAGS "\${CMAKE_CXX_FLAGS} -fsycl -DMKL_ILP64 -qmkl")
```

Reply:
@zhiweij1 Are -qmkl and  -lmkl_sequential -lmkl_core -lmkl_sycl the same thing?
Do we need change -DMKL_ILP64 to -lmkl_intel_lp64?

Reply:
I think `-qmkl=sequential` has same effect as `-lmkl_sycl -lmkl_intel_ilp64 -lmkl_sequential -lmkl_core`

Reply:
If we use `-lmkl_intel_ilp64`, we should add `-DMKL_ILP64`.
If we use `-lmkl_intel_lp64`, we should add `-DMKL_LP64`.